frontend/BoardGUI.py

```diff
@@ -1,18 +1,20 @@
 import pygame as pg
-from backend.Board import Board
-import backend.Cell as CellType
+# from backend.Board import Board
+from backend.GameManager import GameManager
+# import backend.Cell as CellType
 
 vec2 = pg.math.Vector2
 
 class BoardGUI:
-    def __init__(self, game, WIN_SIZE, board_size):
+    def __init__(self, game, WIN_SIZE, board_size, total_pieces):
         self.game = game
         self.board_image = self.get_scaled_image('resources/board/board.png', [WIN_SIZE] * 2)
         self.cell_size = WIN_SIZE // board_size
         self.black_piece_image = self.get_scaled_image('resources/pieces/black_piece.png', [self.cell_size] * 2)
         self.white_piece_image = self.get_scaled_image('resources/pieces/white_piece.png', [self.cell_size] * 2)
-        self.board = Board(board_size)
-        
+        # self.board = Board(board_size)
+        self.backend_game = GameManager(board_size, total_pieces)
+        self.count = 0
```

Comment:
@raspberrymilkyway what is this variable used for?

Reply:
Oh, if that's about the count variable, that's supposed to be commented out -- whoops. There's a line somewhere near that (that's commented out) that prints a count of how many times the clicked function has run next to the description of what's in the clicked cell spot. I must've missed commenting out the counter and its initialization.

Reply:
If it's about the backend_game variable, that's used to handle any backend game management functionality. I can send you an image of the current class interactions if you'd like, but basically it's to separate the backend and the frontend more cleanly -- rather than calling the Board class directly, you'd interact with it through the GameManager class. It also allows the backend to stand and run on its own, which might make testing easier.

Reply:
I'll comment out the count variable and its call when I push the image/example README change.